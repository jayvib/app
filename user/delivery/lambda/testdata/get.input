{
  "created_at": "0001-01-01T00:00:00Z",
  "email": "sanji.vinsmoke@onepiece.com",
  "firstname": "Sanji",
  "id": "uniqueid99",
  "lastname": "Vinsmoke",
  "updated_at": "0001-01-01T00:00:00Z",
  "username": "sanji.vinsmoke"
}