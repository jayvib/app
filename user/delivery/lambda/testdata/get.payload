{
  "body": "eyJ0ZXN0IjoiYm9keSJ9",
  "resource": "/{proxy+}",
  "path": "/user/uniqueid",
  "httpMethod": "GET"
}