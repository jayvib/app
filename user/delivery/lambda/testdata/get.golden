{"id":"uniqueid99","firstname":"Sanji","lastname":"Vinsmoke","email":"sanji.vinsmoke@onepiece.com","username":"sanji.vinsmoke","created_at":"0001-01-01T00:00:00Z","updated_at":"0001-01-01T00:00:00Z"}